apps/editing-toolkit/editing-toolkit-plugin/common/disable-heic-images.js

```diff
@@ -0,0 +1,31 @@
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { createElement } from '@wordpress/element';
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+const DisableHEICImages = createHigherOrderComponent( ( MediaPlaceholder ) => {
+	return ( props ) => {
+		props.onFilesPreUpload = ( files ) => {
+			if ( files ) {
+				Object.values( files ).forEach( ( file ) => {
+					const filename = file.name;
+					const extension = filename.split( '.' ).pop().toLowerCase();
```

Comment:
It just occurred to me that if someone uploads a file that doesn't have a dot, this will throw, as `pop()` would return undefined.

This might be a fix:
```suggestion
					const extension = filename.split( '.' ).pop()?.toLowerCase() ?? '';
```